### ISSUE TITLE
add custom metrics

### ISSUE THREAD

**Issue by fabiofumarola** [fabiofumarola/ultrayolo#15]
Add a custom metric to measure the error on:
- x,y
- w,h
- class
- object loss

https://www.tensorflow.org/api_docs/python/tf/keras/metrics/Metric?version=stable
